packages/schemas/tables/applications.sql

```diff
@@ -12,6 +12,7 @@ create table applications (
   type application_type not null,
   oidc_client_metadata jsonb /* @use OidcClientMetadata */ not null,
   custom_client_metadata jsonb /* @use CustomClientMetadata */ not null default '{}'::jsonb,
+  protected_app_configs jsonb /* @use ProtectedAppConfigs */,
```

Comment:
should be not null?

Reply:
I think it should be nullable:

1. Other type of applications do not need this field.
2. We are unable to genrate a default metadata for other apps since there are some required fields.